spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/util/URLUtil.java

```diff
@@ -149,12 +152,18 @@ private static String removeSlash(String url) {
   private static List<String> getEnvURL(String systemServer) {
     SystemConfiguration sysConfig = new SystemConfiguration();
     EnvironmentConfiguration envConfig = new EnvironmentConfiguration();
-    String sysURL = sysConfig.getString(systemServer);
-    String envURL = envConfig.getString(systemServer);
+    List<Object> sysURL = sysConfig.getList(systemServer);
+    List<Object> envURL = envConfig.getList(systemServer);
     if (StringUtils.isEmpty(sysURL) && StringUtils.isEmpty(envURL)) {
-      sysURL = sysConfig.getString(SYSTEM_KEY_BOTH);
-      envURL = envConfig.getString(SYSTEM_KEY_BOTH);
+      sysURL = sysConfig.getList(SYSTEM_KEY_BOTH);
+      envURL = envConfig.getList(SYSTEM_KEY_BOTH);
+    }
+    List<String> res;
```

Comment:
This method is bit difficult to read. Suggestions:

1. read system property, not exists to step 2, or return
2. read environment, not exists to step 3, or return ...